src/app/producer-shelf/components/action-panels/action-panels.component.ts

```diff
@@ -136,11 +136,11 @@ export class ActionPanelsComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   private updateActionPanels(): void {
-    if (!this.shelfConfiguration || !this.rundown) {
+    if (!this.actionPanelConfigurations || !this.rundown) {
```

Comment:
An empty list is a truthy value, so we can remove this check as it will just result in an empty list for `this.resolvedActionPanels`.

And should we set `this.resolvedActionPanels` to an empty list in this case?

```suggestion
    if (!this.rundown) {
```

Reply:
Changed in: https://github.com/tv2/alba-web-client/pull/456/commits/24e20c1a51870fe40957803e82936792d9ce8c29 to fit the suggestion and i set the list to empty since this is the same approach as for static action and i agree with the approach.